pkg/utils/miscellaneous.go

```diff
@@ -163,6 +169,13 @@ func GetStoreValues(store *druidv1alpha1.StoreSpec) (map[string]interface{}, err
 		"storePrefix":     store.Prefix,
 		"storageProvider": storageProvider,
 	}
+	if strings.EqualFold(string(*store.Provider), Local) {
```

Comment:
I'm wondering whether you should just always try to call getMountPathFromSecretRef(), and if it cannot find the proper data field in the secret, just to not add the storeValues["storageMountPath"] value.

Reply:
I was thinking the same, but I have decided to save one unnecessary call to the Api Server that would be valid only for the local provider and will be unnecessary in 99% of the use cases anyway.

Reply:
Yep thought, you did it because of that :)